src/resources/views/create.blade.php

```diff
@@ -1,64 +1,41 @@
 @extends('backpack::layout')
 
 @section('header')
-	<section class="content-header">
-	  <h1>
-	    {{ trans('backpack::crud.add') }} <span class="text-lowercase">{{ $crud->entity_name }}</span>
-	  </h1>
-	  <ol class="breadcrumb">
-	    <li><a href="{{ url(config('backpack.base.route_prefix'), 'dashboard') }}">{{ trans('backpack::crud.admin') }}</a></li>
-	    <li><a href="{{ url($crud->route) }}" class="text-capitalize">{{ $crud->entity_name_plural }}</a></li>
-	    <li class="active">{{ trans('backpack::crud.add') }}</li>
-	  </ol>
-	</section>
+  <section class="content-header">
+    <h1>
+      {{ trans('backpack::crud.add_a_new') }} <span class="text-lowercase">{{ $crud->entity_name }}</span>
```

Comment:
Good question. 

Before, the title of the main box was "Add a new Job" (for instance) : 
![](https://cloud.githubusercontent.com/assets/1242207/23160190/d1ffe4ac-f825-11e6-9fba-863b891ae5e9.png)

But now, when you have boxes enabled, this box title change :
![](https://cloud.githubusercontent.com/assets/1242207/23139586/6bd4be82-f7ad-11e6-9371-d23f39a703d2.png)

I wanted to keep this "Add a new jobs", so i change this `<h1>`. But this can be discussed.